src/main/java/com/google/devtools/build/lib/rules/java/JavaHeaderCompileAction.java

```diff
@@ -457,6 +457,11 @@ public void build(JavaToolchainProvider javaToolchain) {
                 ExecutionRequirements.REMOTE_EXECUTION_INLINE_OUTPUTS,
                 outputDepsProto.getExecPathString());
       }
+      executionInfo = ImmutableMap.<String, String>builder()
+          .putAll(executionInfo)
+          .putAll(ExecutionRequirements.WORKER_MODE_ENABLED)
+          .putAll(ExecutionRequirements.WORKER_MULTIPLEX_MODE_ENABLED)
+          .build();
```

Comment:
I think we might want the changes here to be behind a flag, similar to other actions that support multiplexed workers